src/components/renderer/form-image.vue

```diff
@@ -18,7 +18,9 @@ export default {
   },
   computed: {
     imageUrl() {
-      return get(this.validationData, this.variableName || this.imageName);
+      if (this.mode === 'preview' ) { 
```

Comment:
This will only work in preview mode. It will always return undefined when you run a process inside a Request Task